### PR TITLE
[14.0][MIG] partner_address_street3

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Comment by kos94ok-3D** [comment]
Depend on: https://github.com/OCA/server-tools/pull/1937

**Review by gurneyalex — APPROVED** [review]

File: partner_address_street3/oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+server-tools https://github.com/akretion/server-tools 14-mig-base_view_inheritance_extension
```
Comment (simahawk) [inline partner_address_street3/oca_dependencies.txt:1]: please drop this :)

**Review by ivantodorovich — APPROVED** [review]

**Review by simahawk — CHANGES_REQUESTED** [review]

File: partner_address_street3/oca_dependencies.txt (hunk 1 above)
Comment (damdam-s) [inline partner_address_street3/oca_dependencies.txt:1]: this has been merged
you can simplify it by pinning oca/14.0 branch

**Review by damdam-s — APPROVED** [review]
a comment on oca_dependencies management

**Review by rousseldenis — APPROVED** [review]
Code review

File: partner_address_street3/oca_dependencies.txt (hunk 1 above)
Comment (simahawk) [reply partner_address_street3/oca_dependencies.txt:1]: actually we should drop the file, we have MQT=PIP

Comment (simahawk) [reply partner_address_street3/oca_dependencies.txt:1]: you can remove the file

**Comment by simahawk** [comment]
merging on #1111

**Pull request closed without merging**